drivers/ina220/include/ina220-regs.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_ina220
+ * @{
+ *
+ * @file
+ * @brief       Register definitions for Texas Instruments INA220 High or Low
+ *              Side, Bi-Directional CURRENT/POWER MONITOR with Two-Wire
+ *              Interface
+ *
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ */
+
+#ifndef INA220_REGS_H
+#define INA220_REGS_H
+
```

Comment:
Some drivers for "simple" sensors define the registers (as there are not too many) in the header file. Just wanted to say.. But I'm fine with this soulution